geniza/corpus/templates/corpus/document_list.html

```diff
@@ -1,17 +1,17 @@
 {% extends 'base.html' %}
-{% load static i18n humanize %}
+{% load static i18n humanize widget_tweaks render_bundle_csp %}
 
 {% block meta_title %}{{ page_title }}{% endblock meta_title %}
 {% block meta_description %}{{ page_description }}{% endblock meta_description %}
 {% block extrascript %}
-    <script defer src="{% static 'js/search.js' %}" nonce="{{ request.csp_nonce }}"></script>
+    {% render_bundle_csp "search" "js" attrs='defer' %}
 {% endblock %}
 
 {% block main %}
     <h1 class="sr-only">{{ page_title }}</h1>
-    <form>
+    <form data-controller="search">
         <fieldset>
-            {{ form.q }}
+            {% render_field form.q data-search-target="query" data-action="input->search#updateSort change->search#submit" %}
```

Comment:
I made a new issue for the search submitting automatically — and I don't think we really want that until we have turbo working; let's disable the `change` handlers for now until we integrate turbo and can apply it to all of the search form fields that require a reload.